packages/theme-chalk/src/tree.scss

```diff
@@ -109,6 +109,9 @@
 
   @include e(label) {
```

Comment:
Replace https://github.com/element-plus/element-plus/blob/dev/packages/theme-chalk/src/tree.scss#L110-#L112 with the following code
```
 @include e(label) {
    width: 100%;
    font-size: getCssVar('font-size', 'base');
 }
```